base/applications/osk/main.c

```diff
@@ -209,15 +219,13 @@ LRESULT OSK_SetKeys(int reason)
                 if (!Keys[i].translate)
                     continue;
```

Comment:
Just to be sure, is it not needed to do `szKey = Keys[i].name;` here, contrary what you have done in a similar code block below?

Reply:
This case is for input language change, to update "translateable" keys, but it doesn't change keys that are not such as Ctrl, Alt etc.